pixel/theme.json

```diff
@@ -251,7 +268,14 @@
             },
             "core/navigation": {
                 "typography": {
-                    "fontSize": "1.125rem"
+                    "fontSize": "0.875rem"
+                },
+                "spacing": {
+                    "blockGap": "1rem",
+                    "padding": {
+                        "left": "0.5rem !important",
```

Comment:
Is the `important` is necessary because the root padding values end up overriding the spacing that is set on the block? Is that a bug, or expected?

Reply:
Without it, navigation inherits root padding leaving more space on the right. There is no way to reset this spacing without adding `!important`

<img width="407" alt="image" src="https://user-images.githubusercontent.com/1935113/188800136-3560448f-fc7c-494a-b4ef-d284be4698ce.png">
<img width="407" alt="image" src="https://user-images.githubusercontent.com/1935113/188800312-ee4b9015-2b01-4409-b86b-2c801690dfde.png">

Created an issue here: https://github.com/WordPress/gutenberg/issues/43931